languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -83779,7 +83779,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
         <rulegroup id="DE_MULTITOKEN_SPELLING_FOUR" name="Rechtschreibfehler in Eigennamen (4 Token)">
             <rule><!--1-->
                 <pattern>
-                    <token postag="UNKNOWN"><exception regexp="yes">&particles_of;|[^\p{L}]</exception></token>
+                    <token postag="UNKNOWN"><exception regexp="yes">&particles_of;|[^\p{L}]|-</exception></token>
```

Comment:
This change and the next one are not needed. Fixed here: https://github.com/languagetool-org/languagetool/commit/2f31080d26632ba822ba932db02d847a076a35c5

Reply:
Thank you!